init.lua

```diff
@@ -222,3 +222,24 @@ minetest.register_chatcommand("whereis", {
 				pos.x, pos.y, pos.z)
 	end
 })
+
+minetest.register_chatcommand("s", {
+    params = "<msg>",
+    description = "Send a message on the staff channel",
+    privs = {kick=true},
+    func = function(name, param)
+    local staff = {}
+    for _, player in pairs(minetest.get_connected_players()) do
+        local toname = player:get_player_name()
+        if minetest.check_player_privs(toname, {kick=true}) then
+            table.insert(staff, toname)
+            minetest.chat_send_player(toname, minetest.colorize("#ff6600",
+                    "«" .. name .. "» " .. param))
+            minetest.log("action", "CHAT[STAFFCHANNEL]: <" .. name .. "> " .. param)
+		if minetest.global_exists("chatplus") then
+			chatplus.log("[STAFFCHANNEL]: <" .. name .. "> " .. param)
+		end
```

Comment:
Lines 239 to 241 can be removed. `chatplus` no longer exists, and it's a waste of effort to add stuff for something that doesn't exist. `ctf_pvp_engine` has quite a few `if chatplus then do something` sorts of code, but only because it was developed in a time when CTF had chatplus and so rubenwardy wanted it to support chatplus-specific features.

Reply:
Done